app/models/train.rb

```diff
@@ -3,5 +3,7 @@
 class Train < ApplicationRecord
   validates :number, presence: true
 
-  belongs_to :railway_station
+  belongs_to :current_station, class_name: 'RailwayStation', foreign_key: :current_station_id
```

Comment:
there is no needs rename DB column if you add  **foreign_key** it also can be 
_belongs_to :current_station, class_name: 'RailwayStation', foreign_key: :railway_station_id
￼Write_

Reply:
Fixed

Reply:
I really happy my comment is become useless.